releasenotes/notes/21715.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+- 25744
```

Comment:
This should be the issue number. https://github.com/istio/istio/issues/21715
```suggestion
- 21715
```

Reply:
Oops...Fixed it.